src/schemas/transaction.yaml

```diff
@@ -224,6 +225,7 @@ TransactionInfo:
         - from
         - hash
         - transactionIndex
+      unevaluatedProperties: false
```

Comment:
Could you explain the rationale for using `unevaluatedProperties` here? My understanding of this keyword is that it is used to allow/disallow properties not covered by pattern matchers or `addtionalProperties`. I believe `additionalProperties` should be used here instead.

Also should not that speccheck [only supports](https://github.com/lightclient/rpctestgen/blob/36ea460705529f6632bbadaba066c3b738e0d499/cmd/speccheck/spec.go#L120-L128) `draft-07` schema and `unevaludatedProperties` was added in `draft-09`. I'm going to look into bumping that, but at the time when I wrote it, the library seemed only support roughly that spec.

Reply:
Using `additionalProperties: false` in this schema throws an error when any key from its `allOf` sibling `oneOf` subschemas is present, while `unevaluatedProperties` "is similar to `additionalProperties` except that it can recognize properties declared in subschemas."

Reply:
made PR to update speccheck with draft 2019-09 - https://github.com/lightclient/rpctestgen/pull/18

Reply:
Thanks for the example! I understand the use of it now and I think your use of it here is correct.